extensions/resteasy-classic/rest-client-config/runtime/src/main/java/io/quarkus/restclient/config/RestClientsConfig.java

```diff
@@ -58,6 +60,7 @@ public interface RestClientsConfig {
      * <p>
      * Can be overwritten by client-specific settings.
      */
+    @WithConverter(TrimmedStringConverter.class)
     Optional<String> proxyAddress();
```

Comment:
```suggestion
    Optional<@WithConverter(TrimmedStringConverter.class) String> proxyAddress();
```

Reply:
Thanks for your hint, .. I am sorry, I am too late, .. geoand has already changed it and merge the pull request into quarkusio:main. Great! If I look after the same annotation & Optional, I find 38x occurrences in the codebase; so theoretical I could refactor them to use the same approach everywhere, could not I.